flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/WindowAggregateTest.scala

```diff
@@ -683,4 +682,165 @@ class WindowAggregateTest extends TableTestBase {
       "must be an integral multiple of step, but got maxSize 3600000 ms and step 1500000 ms")
     util.verifyExplain(sql)
   }
+
+  @Test
```

Comment:
Could you add some tests for other grouping sets syntax, including `CUBE`, `ROLL`.

Reply:
I originally planned to add cases for cube and rollup. However I  skip them because they cannot translate to window aggregate, because there must contain group key which does not contain window_start and window_end.
Do we need to add tests?

Reply:
Can't we support `GROUP BY ROLLUP (a, b),  window_start, window_end`?

Reply:
Sorry for forget those case. Already add tests for `cube`,`rollup`.